### PR TITLE
docs: add not supported devices

### PR DESCRIPTION
**Pull request by herbetom**
I ran accross a few people who tried to use this tool to flash the FRITZ!Box 7362 SL which doesn't seem to work.

### CODE REVIEW

File: docs/locale/de/LC_MESSAGES/index.po [hunk 1]
```diff
@@ -8,14 +8,18 @@ msgid ""
 msgstr ""
```
Comment (mweinelt) [inline docs/locale/de/LC_MESSAGES/index.po:8]: These changes don't seem to belong here.

Comment (herbetom) [reply docs/locale/de/LC_MESSAGES/index.po:8]: I made a separate commit, i could also remove this commit from this PR, but let's see what @blocktrron thinks

**Comment by herbetom** [comment]
@blocktrron what do you think about this?

Note: this is in no way a priority for me, so it doesn't need to be for you either, it just popped up on GitHub

**Comment by blocktrron** [comment]
Let's do this.

**Pull request merged**